implementations/c/source/ockam/transport/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.13)
```

Comment:
other cmake files in the PR set min version to `3.14`, lets make that consistent.